app/lib/actions/katello/host/upload_profiles.rb

```diff
@@ -50,12 +60,13 @@ def run
             Rails.logger.warn("Host with ID %s not found, continuing" % input[:host_id])
           elsif host.content_facet.nil? || host.content_facet.uuid.nil?
             Rails.logger.warn("Host with ID %s has no content facet, continuing" % input[:host_id])
+          elsif profiles.try(:has_key?, "deb_package_profile")
+            payload = profiles.dig("deb_package_profile", "deb_packages") || []
+            import_deb_package_profile(host, payload)
           else
             profiles.each do |profile|
               payload = profile["profile"]
               case profile["content_type"]
-              when "deb"
```

Comment:
I think, we want to keep this as the _new_ interface.

Reply:
I think we can leave it for now since it wouldn't be used until subman sends the proper request format / to the right API per earlier comment. I would like to do this eventually, though.

Reply:
Since we're deprecating the debian profiles API I think I will actually preserve this change.

Reply:
re-added support for this